server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -740,6 +742,122 @@ public boolean accept(File file) {
                             .collect(Collectors.toList());
     }
 
+
+
+    private static File createDir(File parent, String child) throws IOException {
+        File dir = new File(parent, child);
+        dir.mkdirs();
+        return dir;
+    }
+    private static File createDir(String parent, String child) throws IOException {
+        return createDir(new File(parent), child);
+    }
+
+
+    private Map<String, Object> createContext() throws ExtenderException {
+        HashMap<String, Object> empty = new HashMap<>();
+        Map<String, Object> context = context(empty);
+        return context;
+    }
```

Comment:
We did this in a lot of places. Might as well move it to a small helper function